### DISCUSSION TITLE
Clarification: Does TinyGrad use CUDA?

### DISCUSSION THREAD

**Discussion by s4m-mo** [tinygrad/tinygrad#841] [Q&A] [answered]
Just a clarification as I was unsure, does TinyGrad support CUDA for GPU acceleration?

**Comment by jla524** [comment, marked as answer]
It does. You can install dependencies for CUDA by running `pip install -e ".[cuda]"`, and your default device will be set to CUDA.

**Reply by Kripner** [reply]
This has been made obsolete by https://github.com/tinygrad/tinygrad/commit/badc97f824c38a89b10ed37351779fe052976715, which removed the `cuda` extra.
